<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_base3_ud</class>
 <widget class="QWidget" name="Form_base3_ud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Urgency</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="gravity"/>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="datetimeobservation">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="edge">
         <layout class="QGridLayout" name="gridLayout" columnstretch="1,5,1">
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Progression</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Number</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_nombre">
            <property name="minimum">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolButton_calc_nb">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/plugins/Lamia/icons/numpad.png</normaloff>:/plugins/Lamia/icons/numpad.png</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QTextEdit" name="textEdit_evolution">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="node">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stackedWidget_2">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="manhole">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Cover condition</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="conditioncover"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Steps condition</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Culvert condition</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="conditionstep"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Manhole condition</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="conditionmanhole"/>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="conditionculvert"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>H2S suspicion</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QComboBox" name="presenceh2s"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Sedimentation</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QComboBox" name="presencesediments"/>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Pressurized flow</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QComboBox" name="presencepressurizedflow"/>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Maintenance opinion</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QComboBox" name="maintenanceopinion"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="ps">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Sum condition</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="conditionglobal"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Cover condition</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="conditioncover_2"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Floats condition</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="conditioncontroller"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="deshuil">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>General condition</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Sedimentation/clutter</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="presencesediments_2"/>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="conditionglobal_2"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="other">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>General condition</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="conditionglobal_3"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Maintenance opinion</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="maintenanceopinion_2"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="equip">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Equip</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="surfa">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>General condition</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="conditionglobal_4"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="comment">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
